qmerge.c

```diff
@@ -2025,6 +2026,12 @@ int qmerge_main(int argc, char **argv)
 
 	qmerge_strict = contains_set("strict", features) ? 1 : 0;
 
+	binpkg_multi_instance = contains_set("binpkg-multi-instance", features) ? 1 : 0;
+	if (binpkg_multi_instance) {
```

Comment:
Nice to be careful, but let's not do this.  If you insist, pair it with verbose > 1 or something